backend/lib/azimutt_web/admin/event/event_view.ex

```diff
@@ -0,0 +1,64 @@
+defmodule AzimuttWeb.Admin.EventView do
+  # use AzimuttWeb, :view
+  # use Phoenix.View, root: "lib/azimutt_web/admin/templates", path: "*"
+  use Phoenix.View, root: "lib/azimutt_web", namespace: AzimuttWeb
+
+  import Phoenix.HTML.Tag
+  import Phoenix.HTML.Link
+  alias AzimuttWeb.Router.Helpers, as: Routes
+
+  def format_date(date) do
+    {:ok, date_parsed} = Timex.format(date, "{D}/{M}/{YY} à {h24}h{m}:{s}")
+    date_parsed
+  end
+
+  def display_project_name(event) do
+    if event.project !== nil do
+      content_tag(:div, event.project.slug, class: "font-medium")
+    else
+      " -"
+    end
+  end
+
+  def display_organization_name(event) do
+    if event.organization !== nil do
+      event.organization.slug
+    else
+      " - "
+    end
+  end
+
+  def badge(event_name) do
+    default_class = "px-1"
+
+    case event_name do
```

Comment:
What is the logic behind the colors?
Maybe using starts_with or contains can be a bit more generic for new ones?

Reply:
Pour les couleurs, c'est subjectif, j'ai fait un regroupement par type d'actions, on pourras en discuter si tu veux. 
J'ai essayé au début de regrouper par start_with, mais ca rendait illisible car la plupart des events sont des `project_update` ou `project_updated`.